cdap-ui/app/hydrator/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -450,13 +450,14 @@ class HydratorPlusPlusNodeConfigCtrl {
     return true;
   }
 
-  validatePluginProperties(callback) {
+  validatePluginProperties(callback, noSuccessIndicator) {
```

Comment:
nit: don't use double negation to show the success banner. Right now for us to show the success banner we have to pass `false` to this function which is not intuitive. Can you rename the variable to be `showSuccessIndicator` and pass false to hide it?